README.md

```diff
@@ -1,7 +1,236 @@
+<!-- Code generated by "go generate" in github.com/kortschak/gospel; DO NOT EDIT. -->
 # `gospel` [![Build status](https://github.com/kortschak/gospel/workflows/Test/badge.svg)](https://github.com/kortschak/gospel/actions)
 
-The `gospel` program lints Go source code for misspellings in comments and strings. It uses hunspell to identify misspellings and only emits coloured output for visual inspection; don't use it in automated linting.
+The `gospel` program lints Go source files for misspellings in comments,
+strings and embedded files.
+
+It uses hunspell to identify misspellings and makes use of source code
+information to reduce the rate of false positive spelling errors where
+words refer to labels within the source code.
+
 
 ## Installation
 
-Beyond the standard Go installation process, you must also have libhunspell and its header files on your system.
\ No newline at end of file
+Beyond the standard Go installation process, you must also have libhunspell
+and its header files on your system. For a debian-based system this is done
+with `sudo apt install libhunspell-dev`.
+
+
+## Work Flow
+
+`gospel` makes use of module information, and so must be run within a module
+and will only consider source files that are referred to by the module.
+
+`gospel` can be used without configuration in many cases, particularly for
+small source trees where the number of potential errors is likely to be
+small. However, its behaviour can be tuned with [configuration files](#configuration-files).
+The initial setup workflow makes use of the [`.words`](#.words) file to
+build a dictionary of candidate words to ignore.
+
+```
+$ gospel -misspellings=.words ./...
+```
+
+This will collate all the found misspellings in the source tree and write
+them to `.words`. If `gospel` is run again, it will accept the all words in
+the source tree as correctly spelled as they now appear in the projects
+dictionary. Since hunspell dictionaries are just text files, you can go
+through the list of words to remove properly identified misspellings and
+leave in words that are domain-specific and incorrectly marked.
+
+For example the `.words` file at the root of the `gospel` repo includes
+words that would otherwise be flagged with the default "en_US" spelling
```

Comment:
```suggestion
words that would otherwise be flagged with the default `en_US` spelling
```

Reply:
I was in two minds about that; it's a string, not an identifier.

Reply:
Yeah, I get that. It could go either way, but I'd just be consistent about it. I tend to prefer backquotes over double quotes when it's markdown and the text is anything one may type in code or a shell, just because the rendering is nicer.

Reply:
Made consistent.